### ISSUE TITLE
Fails to run if a Gdk package >3.0 exists

### ISSUE THREAD

**Issue by brabes** [lcarsde/menu#1]
Running the application menu fails for me with

```
ImportError: Requiring namespace 'Gdk' version '3.0', but '4.0' is already loaded
```

I'm not a Python expert, but I assume this is because it imports GdkX11 before Gtk, which pulls in a later version of Gdk if it exists.

Fix is to change the import on line 7 to the following (i.e. putting `Gtk` first):

```
from gi.repository import Gtk, GdkX11, GLib
```

**Comment by atennert** [comment]
Thank you very much for the issue report and the fix. I will put it in a little later today and make a new release for it.

**Comment by atennert** [comment]
Fixed with 899ea18d760a1afdc19070997367de7aa92ef299
